kind_ec2_installation.md

```
@@ -0,0 +1,129 @@
+
+
+### 4. Install kubectl
+
+Download **kubectl**, which is a Kubernetes command-line tool.
+
+* Download kubectl binary with curl on Linux
+  ```bash
+  curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+  ```
+
+> **Optional**: **Validate the downloaded kubectl binary  **
```

Markdownlint

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)